packages/mint-components/src/components.d.ts

```diff
@@ -1825,6 +1831,8 @@ export namespace Components {
     }
     interface SqmReferralTableStatusCell {
         "converted": boolean;
+        "fraudStatus"?: "APPROVED" | "PENDING" | "DENIED";
```

Comment:
Consider creating a fraud status type since this literal union is used in a few places.